lib/chewy/type/adapter/orm.rb

```diff
@@ -45,6 +45,7 @@ def identify(collection)
         # Import options:
         #
         #   <tt>:batch_size</tt> - import batch size, 1000 objects by default
+        #   <tt>:direct_import</tt> - use the collectiov values as-is, instead of reloading
```

Comment:
The wording in https://github.com/sideqik/chewy/commit/21df05b77eb8ec72cf09811ea4dcd8919080c4f6 seems more clear to me